tableconvert/src/tableconvert.c

```diff
@@ -338,6 +351,31 @@ void cb_table_convert_type(G_GNUC_UNUSED GtkMenuItem *menuitem, G_GNUC_UNUSED gp
 	convert_to_table(TRUE, GPOINTER_TO_INT(gdata));
 }
 
+void cb_table_convert_change_document(G_GNUC_UNUSED GtkMenuItem *menuitem, G_GNUC_UNUSED gpointer gdata)
```

Comment:
This is a local function so IMHO it should be static and only declared in ```tableconvert.c``` not in ```tableconvert.h```.

Reply:
Should still be static. Unless you prefer it that way.